src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/util/OcrDataDeserializer.java

```diff
@@ -9,45 +9,38 @@
 
 import java.io.IOException;
 import java.util.Base64;
-import java.util.LinkedHashMap;
-import java.util.Map;
 
-import static java.util.stream.Collectors.toMap;
-
-public class OcrDataDeserializer extends StdDeserializer<Map<String, String>> {
+public class OcrDataDeserializer extends StdDeserializer<OcrData> {
```

Comment:
is this custom deserializer still needed, when we use our own class with Jackson annotations??

Reply:
@rkondratowicz I tried not to use custom deserializer but it doesn't work without it, because the ocr_data is Base64 encoded. I tried to find if there is any annotation to specify that that value is in Base64 format, but I couldn't find anything. Any suggestions?

Reply:
oops, forgot it's base 64 encoded

Reply:
missleading class name